src/Asana/AsanaApiClient.php

```diff
@@ -120,6 +130,7 @@ public function createCardsEventOnline(array $values): void
      */
     public function createCardLastMinute(array $values): void
     {
+        $values['titel'] = 'Last Minute: ' . $values['titel'];
```

Comment:
This is not a new issue, but the key should be `title` (in English) for consistency with other keys in `$values`.